master/reference/calicoctl/resources/felixconfig.md

```diff
@@ -71,6 +72,9 @@ spec:
 | metadataPort | The port of the metadata server. This, combined with global.MetadataAddr (if not 'None'), is used to set up a NAT rule, from 169.254.169.254:80 to MetadataAddr:MetadataPort. In most cases this should not need to be changed. | int | int | `8775` |
 | interfacePrefix | The interface name prefix that identifies workload endpoints and so distinguishes them from host endpoint interfaces.  Note: in environments other than bare metal, the orchestrators configure this appropriately.  For example our Kubernetes and Docker integrations set the 'cali' value, and our OpenStack integration sets the 'tap' value. | string | string | `cali` |
 
+
+\* Allowing to run on loose RPF system means containers on the same host can see each other's IP addresses
```

Comment:
I would be more specific here and tell them "Setting `ignoreLooseRPF` to <??> (true or false?), allows containers on the same host to see each other's IP addresses." (<-- notice period at end of sentence)

for such a short note, you could just include this in the table cell itself in my opinion. If it is something you want to call out, you could just preface it with `**Warning**:`

Reply:
I like that sentence. Not a fan of adding warnings in the table itself